From 0de9fdec8ab6891c8c4cd33396bf1ba0946de78a Mon Sep 17 00:00:00 2001
From: "Joshua M. Clulow" <jmc@joyent.com>
Date: Sat, 5 Aug 2017 20:25:44 +0000
Subject: [PATCH] OS-5671 installer should not set "erase" tty option Reviewed
 by: Jason King <jason.king@joyent.com> Approved by: Jason King
 <jason.king@joyent.com>

---
 scripts/prompt-config.sh | 1 -
 1 file changed, 1 deletion(-)

diff --git a/scripts/prompt-config.sh b/scripts/prompt-config.sh
index bed13be2..6751830a 100755
--- a/scripts/prompt-config.sh
+++ b/scripts/prompt-config.sh
@@ -819,7 +819,6 @@ done
 updatenicstates
 
 export TERM=xterm-color
-stty erase ^H
 
 trap sig_doshell SIGINT
 
-- 
2.21.0

